src/domain/account/entities/__tests__/verification-code.builder.ts

```diff
@@ -0,0 +1,10 @@
+import { Builder, IBuilder } from '@/__tests__/builder';
+import { VerificationCode } from '@/domain/account/entities/account.entity';
+import { faker } from '@faker-js/faker';
+
+export function verificationCodeBuilder(): IBuilder<VerificationCode> {
+  return new Builder<VerificationCode>()
+    .with('code', faker.string.numeric({ length: 6 }))
+    .with('generatedOn', new Date())
```

Comment:
Should we randomize this date value at builder level?

Reply:
For this one I think it's better to not randomize it as most code generations should take the current `Date` value as a default. Testing a value that does not represent the "now" seems to be less likely than one with the current value. Additionally the goal of the builders is to reduce verbosity in the tests with "sensible" defaults.